contributions/catalog/webapps/templates/contribute/talent_details.html

```diff
@@ -9,7 +9,7 @@
     <link href="{{ url_for('static', filename='css/style.css') }}" rel="stylesheet" type="text/css">
     <link href="{{ url_for('static', filename='css/bar.css') }}" rel="stylesheet" type="text/css">
     <link href="{{ url_for('static', filename='js/modernizr.js') }}" rel="javascript">
-    <script src="../../static/js/modernizr.js"></script> <!-- Modernizr -->
+    <script src="../../../static/js/modernizr.js"></script> <!-- Modernizr -->
     <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
```

Comment:
In Talent details page, for Self certification, if only one field is present, the spacing is not right. Not related to this PR, but I just noticed it.
![Screen Shot 2022-04-25 at 9 25 42 AM](https://user-images.githubusercontent.com/22112888/165109629-de65a16a-0427-4873-a44b-043718652aff.png)

Reply:
can you create an issue for this?

Reply:
[Issue](https://github.com/rokwire/rokwire-building-blocks-api/issues/913) created